db/migrate/20231004211307_add_devise_to_users.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+class AddDeviseToUsers < ActiveRecord::Migration[7.0]
+  def self.up
+    change_table :users do |t|
+      ## Database authenticatable
+      t.string :email,              null: false, default: ""
+      t.string :encrypted_password, null: false, default: ""
+
+      ## Recoverable
+      t.string   :reset_password_token
+      t.datetime :reset_password_sent_at
+
+      ## Rememberable
+      t.datetime :remember_created_at
+
+      ## Trackable
+      # t.integer  :sign_in_count, default: 0, null: false
+      # t.datetime :current_sign_in_at
+      # t.datetime :last_sign_in_at
+      # t.string   :current_sign_in_ip
+      # t.string   :last_sign_in_ip
+
+      ## Confirmable
+      # t.string   :confirmation_token
+      # t.datetime :confirmed_at
+      # t.datetime :confirmation_sent_at
+      # t.string   :unconfirmed_email # Only if using reconfirmable
+
```

Comment:
-  You have done great so far :+1: but there are still some requirements you need to make before moving to the next milestone. :+1: :+1: 
- [ ] Looking at the video, you can see that the `rails db:migrate` is failing. there is an error in the devise migration. this is because according to the requirement, you are meant to enable `:confirmable` which you have dome in the `user` model. but for that to work, you need to also add the necessary columns to the `Users table` . to fix this issue you need to uncomment from line `25 to 28`. this is going to fix the migration issue. :+1: :+1: :wave: :smile: 

```ruby
      t.string   :confirmation_token
      t.datetime :confirmed_at
      t.datetime :confirmation_sent_at
      t.string   :unconfirmed_email # Only if using reconfirmable
```

- - Video reference:

https://github.com/alexmedinasf/Blog/assets/107632780/fa7fbb1c-e6a7-4450-b335-705023efc283

Reply:
Thank you for the review, @V-Blaze , I'll work on this changes right now!